web/src/lib/utils/focus-util.ts

```diff
@@ -12,7 +12,7 @@ export const setDefaultTabbleOptions = (options: TabbableOpts) => {
 export const getTabbable = (container: Element, includeContainer: boolean = false) =>
   tabbable(container, { ...defaultOpts, includeContainer });
 
-export const focusNext = (selector: (element: HTMLElement | SVGElement) => boolean, forwardDirection: boolean) => {
+export const moveFocus = (selector: (element: HTMLElement | SVGElement) => boolean, forwardDirection: boolean) => {
```

Comment:
`forwardDirection: boolean` is rather ugly and anything but explicit. I'd strongly vote for just `direction: 'previous' | 'next'